### PR TITLE
Javascript/javascript1/week2

### PR DESCRIPTION
**Pull request by soujanyapolavarapu**

### CODE REVIEW

File: javascript/javascript1/week2/Candy-helper.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+addCandy('Toffee', 2);
+addCandy('Chewing-gum', 18);
+console.log(boughtCandyPrices);
+for (i=0; i<boughtCandyPrices.length; i++){
+   totalPrice= totalPrice + boughtCandyPrices[i];
+}
+console.log('Total price of the 1st Array ' + totalPrice);
+console.log('Total amount that we spent for candies is ' + amountToSpend);
+canBuyMoreCandy( );
+if(true){
```
Comment (aina21) [inline javascript/javascript1/week2/Candy-helper.js:19]: do you need this condition?

File: javascript/javascript1/week2/Candy-helper.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+else
+{
+  console.log("Enough candy for you!");
+}
+
+
+
+
+function addCandy (candyType , weight) {
+  if(candyType== 'sweet'){
```
Comment (aina21) [inline javascript/javascript1/week2/Candy-helper.js:31]: maybe the switch is more readable

File: javascript/javascript1/week2/Event-application.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+
+  let date = new Date();
```
Comment (aina21) [inline javascript/javascript1/week2/Event-application.js:2]: const

File: javascript/javascript1/week2/Event-application.js [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+  let today = date.getDay();
```
Comment (aina21) [inline javascript/javascript1/week2/Event-application.js:3]: always use const until it doesn't let you

File: javascript/javascript1/week2/Student-manager.js [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+addStudentToClass('Rajesh');
+addStudentToClass('Student5');
+addStudentToClass('Student6');
+addStudentToClass('Student7');
+addStudentToClass('Queen Margrethe');
+getNumberOfStudents();
+console.log(class07Students);
+
+function addStudentToClass(studentName) {
+  if (studentName === 'Queen Margrethe'){
```
Comment (aina21) [inline javascript/javascript1/week2/Student-manager.js:16]: what happened if you add queen twice?

**Review by aina21 — CHANGES_REQUESTED** [review]
good job 👍

File: javascript/javascript1/week2/Event-application.js (hunk 4 above)
Comment (soujanyapolavarapu) [reply javascript/javascript1/week2/Event-application.js:3]: ok

File: javascript/javascript1/week2/Event-application.js (hunk 3 above)
Comment (soujanyapolavarapu) [reply javascript/javascript1/week2/Event-application.js:2]: ok

File: javascript/javascript1/week2/Student-manager.js (hunk 5 above)
Comment (soujanyapolavarapu) [reply javascript/javascript1/week2/Student-manager.js:16]: yes got it

File: javascript/javascript1/week2/Candy-helper.js (hunk 2 above)
Comment (soujanyapolavarapu) [reply javascript/javascript1/week2/Candy-helper.js:31]: I will try

File: javascript/javascript1/week2/Student-manager.js (hunk 5 above)
Comment (soujanyapolavarapu) [reply javascript/javascript1/week2/Student-manager.js:16]: Thankyou so much for your review..... :)

**Pull request merged**